api/src/services/report/textforms-parser.js

```diff
@@ -256,8 +256,8 @@ exports.isCompact = (def, msg, locale) => {
   }
   const labels = _.flatten(_.map(_.values(def.fields), field => {
     return [
-      config.translate(field.labels.tiny, locale),
-      config.translate(field.labels.short, locale)
+      config.translate(field.labels && field.labels.tiny, locale),
```

Comment:
according to the docs labels aren't required and so this would crash without the intermediate check